README.md

```diff
@@ -3,88 +3,85 @@
 This repository contains a demo application to show how [Broadway] can be used within a Symfony application.
 The example is taken from the [Practical Event Sourcing][practical-eventsourcing] talk from [Mathias Verraes].
 
-[Broadway]: https://github.com/qandidate-labs/broadway
+[Broadway]: https://github.com/broadway/broadway
 [practical-eventsourcing]: http://verraes.net/2014/03/practical-event-sourcing.markdown/
 [Mathias Verraes]: https://twitter.com/mathiasverraes
 
-## Installation
+For simplicity the demo uses the official [DBAL event store] and a custom DBAL read model implementation.
+You will need to have [SQLite] installed to run the demo.
 
-### Vagrant
+[DBAL event store]: https://github.com/broadway/event-store-dbal
+[SQLite]: https://www.sqlite.org/
 
-```sh
-vagrant up
-```
-
-Vagrant will provision the machine (using ansible).
-
-The box is then available at `192.168.42.100`, the vhost is configured with domain `broadway.local`.
-
-If you don't have ansible installed, you will have to run `composer install` yourself. And run `app/console broadway:event-store:schema:init` to initialize the database.
-
-### Docker Compose
+## Running the demo
 
-```sh
-docker-compose up -d
 ```
-
-Docker-compose will set up the containers needed to run this demo.
-
-
-```sh
-docker-compose exec app /bin/bash
-app/console broadway:event-store:schema:init
+composer install
+bin/console broadway:event-store:schema:init
```

Comment:
`bin/console doctrine:database:create`?
But there is no way to create the schema right now... :crying_cat_face:

Reply:
there are commands in this repo: https://github.com/othillo/broadway-demo/blob/49b8139a35d47ecd61fb0e5c599dfc5222f7deae/src/Command/CreateReadModelCommand.php#L41

Reply:
But only for the readmodels, not for the eventstore?

I noticed after my comment that the doctrine command only creates the sqlite file, nothing else. So pretty pointless :D

Reply:
see https://github.com/broadway/broadway-bundle/issues/54#issuecomment-397655283